tcms/templates/run/get_case_runs.html

```diff
@@ -156,20 +156,15 @@
 				{{/each}}
 			<div class="bug_type" id="id_bug_type">Issue Tracker</div>
 			<select name="bug_system_id" id="bug_system_id">
-				<option value="1" selected="selected" data-tab="bug-system-bugzilla">Bugzilla</option>
-				<option value="2" data-tab="bug-system-jira">JIRA</option>
+				{{#each bugTrackers}}
+					<option value="{{ @key }}">{{ this }}</option>
+				{{/each}}
 			</select>
 			</br>
-			<div id="bug-system-bugzilla" style="line-height:30px;">
+			<div id="bug-system" style="line-height:30px;">
 				<div class="bug_id">Bug ID</div>
 					<input type="text" name="bug_id">
 					</br>
-					<input type="checkbox" name="bz_external_track" value="bz_external_track">&nbsp;
-					<span class="use_bz_external_track">Check to add Test Cases to BZ</span>
```

Comment:
Please bring the tabs and checkbox ``Check to add Test Cases to BZ`` back.

I agree that the checkbox could be configurable, but at this moment, it should be still usable.

Tabs are also useful when there are a few bug trackers enabled. Different bug trackers can have their own settings and display corresponding controls in the dialog for users, that would be flexible to add more controls in the dialog in the future.

I think you could just fix this commit instead of making another commit.